Added saved filters section (#144)
spacelift-io/user-documentation: docs/concepts/stack/organizing-stacks.md

## Saving filters in views

It is possible to save your filters with a Filters Tab. You can select all the filters that you would like to apply, add search query or sorting in the top right corner, click New View, enter the name of the view and click Save. This view is now saved for this account. During creation you can also mark your new view as your personal default view. Next time you login or navigate to stacks your personal default view will be applied.

![](../../assets/screenshots/saved-views-filters-tab.png)

If you forgot to mark your view as default then you can easily do the same thing in the Views Tab.

![](../../assets/screenshots/saved-views-default.png)

### Shared views

Views can be shared or private. While first creating the view, it is available only for your user. If you have an admin access you can make it public for all the users of the account by hovering over the saved view and clicking small eye icon "Share within account". This way all the users within this application can see the saved view and who created it.

### Reseting

> If you want to quickly reset your default view to Spacelift default state, click "Reset to Spacelift default view" button. It will result in clearing all sorting, search and filter parameters, as well as manage filters settings.

Reviewer: ```suggestion
To quickly reset your default view to Spacelift default state, click the "Reset to Spacelift default view" button. It will result in clearing all sorting, search, and filter parameters, as well as managed filter settings.
```
